src/streamlink_cli/utils/lazy_formatter.py

```diff
@@ -0,0 +1,21 @@
+import string
```

Comment:
This is in the wrong module :) It should be in `src/streamlink/utils/lazy_formatter.py`.